.github/workflows/release.yml

```diff
@@ -139,21 +136,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.ORG_GORELEASER_GITHUB_TOKEN }}
           COSIGN_PWD: ${{ secrets.ORG_COSIGN_PWD }}
       # Trigger installation tests in otel-collector-installer-testing
-      - name: Get Tag From Environment
-        id: get-tag
-        run: printf '::set-output name=tag::%s' "$(printf '%s' "${{ github.ref }}" | sed 's/refs\/tags\///')"
       - name: Trigger Installation Testing
         uses: peter-evans/repository-dispatch@v2
         with:
           token: ${{ secrets.ORG_GORELEASER_GITHUB_TOKEN }}
           repository: observIQ/otel-collector-installer-testing
           event-type: upstream_prerelease
-          client-payload: '{ "version": "${{ steps.get-tag.outputs.tag }}" }'
+          client-payload: '{ "version": "$GITHUB_REF_NAME" }'
```

Comment:
This doesn't look like it runs in a shell, does this need to be `${{ github.ref_name }}` instead of "$GITHUB_REF_NAME"?

Same with the `Update Amplify Version` step.

Reply:
It's funny you ask I had that before because I wasn't sure. If you're also second guessing it I'll switch it back